CMakeLists.txt

```diff
@@ -97,14 +97,18 @@ if(PRECOMPILED_DIR)
     ${PRECOMPILED_DIR}/pcre.lib)
 endif()
 
-# COLLADASaxFrameworkLoader/BaseUtils/Framework, GeneratedSaxParser
-include_directories(dependencies/OpenCOLLADA/OpenCOLLADA/COLLADASaxFrameworkLoader/include)
-include_directories(dependencies/OpenCOLLADA/OpenCOLLADA/COLLADABaseUtils/include)
-include_directories(dependencies/OpenCOLLADA/OpenCOLLADA/COLLADAFramework/include)
-include_directories(dependencies/OpenCOLLADA/OpenCOLLADA/GeneratedSaxParser/include)
-if(NOT OpenCOLLADA)
-  add_subdirectory(dependencies/OpenCOLLADA/modules/COLLADASaxFrameworkLoader)
-  set(OpenCOLLADA COLLADASaxFrameworkLoader)
+if(NOT DEFINED COLLADA_LIBRARIES AND NOT DEFINED COLLADA_FRAMEWORK_LIBRARIES)
```

Comment:
While I'm not opposed to build changes allowing specifying alternate locations for libraries as in #125, please remove this and open a separate pull request for it.